### PR TITLE
use safe loader wrapper to isolate use.

### PR DESCRIPTION
**Pull request by kdaily**
*Issue #, if available:*

*Description of changes:*

This addresses the underlying cause of tests for the ECS customization failing. The cloudformation customization is changing global options for PyYAML's safe loader. A solution for this is provided here:

https://github.com/yaml/pyyaml/issues/221

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: awscli/customizations/cloudformation/yamlhelper.py [hunk 1]
```diff
@@ -80,6 +80,9 @@ def _dict_constructor(loader, node):
     return OrderedDict(loader.construct_pairs(node))
 
 
+class SafeLoaderWrapper(yaml.SafeLoader):  # type: ignore
```
Comment (vz10) [inline awscli/customizations/cloudformation/yamlhelper.py:83]: Let's add some comment do we need this because it's not clear without full context.

**Review by vz10 — CHANGES_REQUESTED** [review]
Looks good. Just a small comment

File: awscli/customizations/cloudformation/yamlhelper.py (hunk 1 above)
Comment (vz10) [reply awscli/customizations/cloudformation/yamlhelper.py:83]: And let's remove "# type: ignore" because we don't check typing anyway

Comment (kdaily) [reply awscli/customizations/cloudformation/yamlhelper.py]: Done!

**Review by vz10 — APPROVED** [review]
Thank you 🚢

**Review by mgrandis — APPROVED** [review]

**Pull request merged**